docs/deployments/deploy-chrome-extension.mdx

```diff
@@ -1,79 +1,73 @@
 ---
-title: Deploy a Chrome Extension
-description: Learn how to deploy a Chrome Extension with Clerk.
+title: Deploy a Chrome Extension to production
+description: Learn how to deploy a Chrome Extension to production with Clerk.
 ---
 
 ## Create a production instance
```

Comment:
Should there be a note like on some of the other pages that these instructions are for Plasmo, but the customer can modify if they are using a different framework?

Reply:
```suggestion
> [!WARNING]
> This guide assumes you are using the Plasmo framework. If you need help applying these concepts to another framework, [contact support](https://clerk.com/contact/support).

## Create a production instance
```